lincbrain/dandiapi.py

```diff
@@ -417,7 +417,7 @@ def __init__(
         :param str token: User API Key. Note that different instance APIs have
             different keys.
         """
-        check_dandi_version()
```

Comment:
`check_dandi_version()` (and `versioneer` by extension) still need to be set up correctly -- thus references are commented out for now.